package/osx/fix-arm64-library-paths.R

```diff
@@ -0,0 +1,47 @@
+
```

Comment:
Should this be an executable R script w/ shebang?

Reply:
Done!